### PR TITLE
Converted Navbar Component from ANT Design to Material UI

### LINKED ISSUE #89: Convert Navbar Component from ANT Design to Material UI
replace ant design with material-ui design in navbar component

### PR DESCRIPTION
**Pull request by rijusougata13**
Convert Navbar Component from ANT Design to Material UI
## Description

Replaced all ant design with material ui component
## Related Issue
Fixed #89 

## Screenshots (In case of UI changes):
# Desktop view (new user)
Screenshot from 2021-05-29 15-49-29
# Desktop view (after signin)
Screenshot from 2021-05-29 15-38-32
# Mobile view
Screenshot from 2021-05-29 15-38-48

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by Niweera — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -69,7 +69,7 @@
     "@storybook/node-logger": "^6.2.9",
     "@storybook/preset-create-react-app": "^3.1.7",
     "@storybook/react": "^6.2.9",
-    "babel-loader": "^8.2.2",
+    "babel-loader": "^8.1.0",
```
Comment (shehand) [inline package.json:72]: Any specific reason to downgrade?

File: src/components/NavBar/MainNavbar/RightMenu.js [hunk 2]
```diff
@@ -1,21 +1,32 @@
...
-  CodeOutlined,
-  LogoutOutlined,
-  BlockOutlined,
-  SettingOutlined
-} from "@ant-design/icons";
+import BlockOutlinedIcon from "@material-ui/icons/BlockOutlined";
+import SettingsOutlinedIcon from "@material-ui/icons/SettingsOutlined";
+import CodeOutlinedIcon from "@material-ui/icons/CodeOutlined";
+import ExitToAppOutlinedIcon from "@material-ui/icons/ExitToAppOutlined";
+import PersonOutlineOutlinedIcon from "@material-ui/icons/PersonOutlineOutlined";
```
Comment (shehand) [inline src/components/NavBar/MainNavbar/RightMenu.js:11]: I'm not sure this is a good thing to do. Having more imports for icons makes code messier. Can't we follow the method we used to follow here?

**Review by shehand — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (rijusougata13) [reply package.json]: got some dependencies error for that version,  but later i realized the problem is in another package . sorry, forgot to revert it back. changing the version as it was before.

File: src/components/NavBar/MainNavbar/RightMenu.js (hunk 2 above)
Comment (rijusougata13) [reply src/components/NavBar/MainNavbar/RightMenu.js:11]: it was discussed in https://github.com/scorelab/Codelabz/pull/75#discussion_r638543643. So I was following this method.

**Review by shehand — APPROVED** [review]

**Comment by rijusougata13** [comment]
@thisuraseniya  Its ready to be merged.

**Pull request merged**